src/generic_layers/reduced_points.rs

```diff
@@ -58,7 +58,7 @@ impl<P: Reducible, const SIZE: u8, const SALT: u64> Chunk for ReducedUniformPoin
             .points
         {
             for other in raw_points.get_range(Bounds {
-                min: p.position(),
+                min: p.position() - Point2d::splat(p.radius()),
                 max: p.position() + Point2d::splat(p.radius()),
             }) {

```

Comment:
You can use `Bounds::point(p.position).pad(Point2d::splat(p.radius()))` to get this behaviour.

I want to make this simpler, as it's such a common operation, so I may just add a `pad` method to `Point2d`